site/layouts/pages/hub.html

```diff
@@ -0,0 +1,184 @@
+              </div>
+            </div>
+
+            <div class="portrait-small__content">
+              <div class="portrait__name h4-like">
+                <a href="/#" class="title">Camille Exemple</a>
+              </div>
+            </div>
+          </div>
+
+
+
+          <div class="hub--video">
+            {{ .Content }}
+          </div>
+        </section>
+
+        <section class="hub--box" id="programme">
+          <h2 class="no-top-margin">Programme</h2>
+
+        </section>
+
+        <section class="hub--box" id="partenaires">
+          <h2 class="no-top-margin">Partenaires</h2>
+
+        </section>
+
+        <section class="hub--box" id="communaute">
+          <h2 class="no-top-margin">Communauté</h2>
+          <p class="big">La force de Paris Web depuis 15 ans, c'est vous, la communauté. Pour vous permettre d'échanger à tout moment de l'événement et même au-delà, nous vous proposons deux espaces d'échange.</p>
+        </section>
+      </div>
+      
+    </div>
+  </main>
+
+  <footer class="footer--hub">
+    <div class="logo">
+      <img src="{{.Site.BaseURL}}images/logo/2019/logo-rond.svg" alt="" />
+    </div>
+    <div class="text">
+      <p class="small">Depuis 2006, Paris Web est le rendez-vous incontournable des professionnels francophones du Web. Les conférences et ateliers Paris Web sont organisés par l’association à but non lucratif Paris-Web.</p>
+    </div>
+    <div>
+      <ul class="link-list">
+        <li>
+          <a href="#">Qui sommes-nous ?</a>
+        </li>
+        <li>
+          <a href="#">Nous contacter</a>
+        </li>
+        <li>
+          <a href="#">Le site paris-web.fr</a>
+        </li>
+      </ul>
+    </div>
+    <div>
+      <ul class="link-list">
+        <li>
+          <a href="#">Aide</a>
+        </li>
+        <li>
+          <a href="#">Code de conduite</a>
+        </li>
+        <li>
+          <a href="#">Partenaires</a>
+        </li>
+      </ul>
+    </div>
+  </footer>
+
+{{ partial "footer" . }}
```

Comment:
(Retour à la ligne en fin de fichier.)

```suggestion
{{ partial "footer" . }}
```